### PR TITLE
feat: support for optional form

### PR DESCRIPTION
**Pull request by douglaseggleton**
Removes the required 'form' prop so that Embed can be used without a form.

### CODE REVIEW

File: packages/embed-react/README.md [hunk 1]
```diff
@@ -158,10 +158,30 @@ Embed will automatically submit the payment form with hidden inputs, this can be
...
+
+const MyComponent = () => {
+  const embed = useRef<EmbedInstance>();
+
+  return <form>
+    <Gr4vyEmbed ref={embed} {...}/>
+    <button onClick={() => embed.current.submit()}>
+  </form>
+
+}
```
Comment (cbetta) [inline packages/embed-react/README.md:177]: This example is a tad confusing. You say a `form` is not required but then you use a `form` in the example. Maybe replace it with a `div`?

Comment (douglaseggleton) [reply packages/embed-react/README.md:177]: It's still good practice to use a form for semantics (a11y) - its just we won't hijack the form and take over the submit.
Happy to change to a div though.

File: packages/embed-react/README.md [hunk 2]
```diff
@@ -158,10 +158,30 @@ Embed will automatically submit the payment form with hidden inputs, this can be
 />
 ```
 
+### Submission without a Form
+
+Embed does not require a form to be present, in order to access the `EmbedInstance` directly you can pass a ref.
```
Comment (gryevns) [inline packages/embed-react/README.md:163]: Should we mention `onComplete` here?

**Review by gryevns — APPROVED** [review]

**Comment by gr4vy-code** [comment]
:rocket: PR was released in [`v2.4.0`](https://github.com/gr4vy/gr4vy-embed/releases/tag/v2.4.0) :rocket:

**Pull request merged**